ops/terraform/env/mgmt/base_config/values/mgmt.yaml

```diff
@@ -48,6 +48,6 @@ bfd:
     cpm:
       cpm_assume_role_arn: <<CIPHER>>AgV48tBwYlzigYl1rkp0ahbcgEK0qoEZDM5RLWUQHfhLCAgAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREF2VCtZa2dnVFIwWW51ejZzL1krSzJkZm5hdWdGWnlDMHA1L3V3WWI5OEI2cGVpdmdjNDdxSHBmZ0M5ZXZ5S25Wdz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwFud60SeDeO73Qp19-PNh_kAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMN4Xu-w9MEq6th6PiAgEQgDsBMkuVbeBFbZD5MbYx24HWvP_ddnmVQWILwR8SPjc_Fo6TxOQMD_9KlGUzzIBwVqqQ3PVDVkpU2rcgSgIAABAAB8gfLvrDX05x9S1oY74-F_K7sOwQgOn6e-0BdHHgjIS9KQs6IrmXEebzdQuPL8OY_____wAAAAEAAAAAAAAAAAAAAAEAAABRvnoLieiV7B-Bae7bVa5Zb4m2BnPGkR77q4p77pwTpw4cwWqy-DUzBmqe_Q1Y9zMSOrba6NEswdiQLOelkwoosEiZKwrKoS0UWv4l6YKZ202dmyPPROh1CzTKSSupLEg3oQBnMGUCMFs5X1lx_F9kCoOojg-xYahOQHYctmZNP55SH6XbCg9z28Ayrlo8voQU4WElrsWYSAIxANAu2UwDou1li2Nxwv5UpXLJ5uddH83PslCxFhmoE7YithRUoLNx1flfZPz6UOSNfQ==<</CIPHER>>
-      cpm_aws_account_arn: <<CIPHER>>AgV4HDHebiV7RfGqDhG818QahYI-TMojmgjjJvZgwT0oROMAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFrMU8wTXpHekJxNXdtUEpZeW1XTUdlVmV6NnVLOEx4NGtTVlpaU0hJQk8xU3Niclh6eDdibGpITG5hekVaQTBRZz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwGvCnxpzrKsdq9vImFb2WeUAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQM4sTmn9CMBL6NpxU_AgEQgDvGoJ80oghWOzV8CL_CIrG0R66cZUDZn9tAAvrftNATKWzxGzK_ieP2whKxM86fmYkuvL0zB6oYQ4ZRJAIAABAAtx218tRWbzc5LWo4hgZDeYZqloQ6itcRa26D3Yi_IxMPFfG0zzMc7629oby9ych9_____wAAAAEAAAAAAAAAAAAAAAEAAAAgaabmA_g2C470ipdswxIVwtwCwtCQKHl7Z2faWnBobOnvSGUK-P3iTeE2wU6I5G0QAGcwZQIwLzYBA_lDua76W9MzVD9u4vZfcD-P29fTFaVZBPupBG2R2XiQyFO-_NUODLWpq6SaAjEA8cpdy6YZf-Mg9LvWNOPvFXuwYwlPxz22GFg-vO6ZUlfNz72sdxM5rsHM5jDBVezv<</CIPHER>>
+      cpm_aws_account_arn: <<CIPHER>>AgV4uO1Gk4qR7y5fuMinS-89sH-rvubGMfsbLt9RfWIeG7YAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFtMjBSUE9TQm1jUU1DY1JPc2xZN3VVWXV6NURKUFZPa1NhbUhDQ2JZVDdWbEpzd1FucC9yckE1TThSTXF6MkhKQT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwFQxQtnLPhfLaTX9WYoiFsXAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMwc5l8NFpiTJjX-SQAgEQgDuYmLSpWTvJ0x52w1y7BlOuEZ2zneDa507e88E4hlmxGepLuD26IOJTyOshIQkEla-w_ZvACHnZVTR1HgIAABAAACovcmta-PG0AXBT4gzRKLzUsAZgZYErbVxjWn5huTN8Yn7hSvAQKW9nJPGl00lP_____wAAAAEAAAAAAAAAAAAAAAEAAAAgpzWRgfzpfo0KDAf3FeaaGYBnGrcOMR9iT1GyEASrFDYmg9TLX8fKaBg6o5NABMIZAGcwZQIwbHQR2niUeP1bkDRabQbjTi6o7HAzvTLa6V0ffFEAu9dVtjwRAL8gLfBAQIqsZlpzAjEAlQD-Cy7twHzUXAbWPm2bY0BEMz7MC4jV6ul0ozs0f8gvU59nVZ2QvmwwGnA2daHs<</CIPHER>>
```

Comment:
Is this change related?

Reply:
Oh, no they are not. I'm not sure why that value gets re-ciphered on my machine as there are no changes to the file. I diffed this branch with the trunk and there are no changes to the unciphered values. Either way I'll remove that change now.

Reply:
My guess is trailing whitespaces that my editor automatically fixes.